### PR TITLE
chore: update `cache-customization` section

### PR DESCRIPTION
**Pull request by TetieWasTaken**
**Please describe the changes this PR makes and why it should be merged:**
Mostly semantic changes. But also a change which I was not too sure about.

On this line (and 1 similar one later)
```js
keepOverLimit: (member) => member.id === client.user.id
```
I'd change it to
```js
keepOverLimit: (member) => member.id === YourClientID,
```

I'm assuming this is the correct option since in the caching options your client isn't logged in yet (i.e `client.user` is undefined). But I'm not sure if this might be done internally in the `Sweepers`? Although I'm not convinced that happens since the filter just gets passed on. But please do still correct me if I'm wrong here.

https://github.com/discordjs/discord.js/blob/6a221a967642d431bed80f6b09c40021d278f26c/packages/discord.js/src/util/Sweepers.js#L97

### CODE REVIEW

File: guide/miscellaneous/cache-customization.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Cache customization
 
-Sometimes, you would like to be able to customize discord.js's caching behavior in order to reduce memory usage.
-To this end, discord.js provides you with two ways to do so:
+Sometimes, you might want to customize the caching behavior of discord.js.
```
Comment (jaw0r3k) [inline guide/miscellaneous/cache-customization.md:3]:
```suggestion
Sometimes, you might want to customize the caching behavior of discord.js in order to reduce memory usage.
```

This part was useful

Comment (TetieWasTaken) [reply guide/miscellaneous/cache-customization.md:3]: Oops, missed that.

File: guide/miscellaneous/cache-customization.md [hunk 2]
```diff
@@ -53,12 +53,16 @@ const client = new Client({
 		ReactionManager: 0,
 		GuildMemberManager: {
 			maxSize: 200,
-			keepOverLimit: member => member.id === client.user.id,
+			keepOverLimit: (member) => member.id === YourClientID,
```
Comment (jaw0r3k) [inline guide/miscellaneous/cache-customization.md:56]: this was working

**Comment by almostSouji** [comment]
this is a non-issue since sweepers get evaluated after login when `client#user` is present
if there are some typos, but semantic changes are not necessary or improve reading flow.

**Pull request closed without merging**